api/model-server.yaml

```diff
@@ -0,0 +1,607 @@
+# WARNING - EXPERIMENTAL
+# This file was auto generated from the existing API using an IntelliJ plugin,
+# see https://www.jetbrains.com/help/idea/openapi.html#generate_openapi
+#
+# Manual changes were done for this 'spec' to work in-place with the
+# model-server for now. A lot of changes were done and are still necessary
+# to make this OpenAPI a viable artifact. It will most likely be split
+# into multiple OpenAPI files.
+
+openapi: "3.0.3"
+info:
+  title: "model-server public API"
+  description: "modelix public API"
+  version: "1.0.0"
+servers:
+  - url: '/'
```

Comment:
```suggestion
  - url: '/v2'
```

Reply:
We need to follow up on this. I created an issue here to separate this discussion from this PR.

[MODELIX-683](https://issues.modelix.org/issue/MODELIX-683) Decide and organize model-server endpoints (via OpenAPI definition)